### ISSUE TITLE
Mimic RACL like cross attention between tasks

### ISSUE THREAD

**Issue by pmhalvor** [pmhalvor/fgsa#18]
- [x] build RACL like architecture in PyTorch
  - [x] L2 normalization
  - [x] correct queries, keys, values into attention blocks
  - [x] build each of the 4 relation types in forward
- [x] check model is learning
- [x] hyper parameter search
- [x] comparable to TensorFlow-RACL (OG)?
